### PR TITLE
Api key

### PR DESCRIPTION
**Pull request by nadazaric**
Added Api Keys to Madagascar.

### CODE REVIEW

File: zanzibar/app.py [hunk 1]
```diff
@@ -48,5 +49,16 @@ def delete_acl_entry():
 
     return jsonify({'message': 'ACL entry deleted successfully', 'entry': acl_entry.to_dict()}), 200
 
+
+@app.route('/api-key', methods=['GET'])
+def get_app_key():
+    app_key = generate_api_key()
+    return jsonify({'message': 'Api-Key created.', 'api-key': app_key}), 201
+
+@app.route('/dummy', methods=['GET'])
```
Comment (tince250) [inline zanzibar/app.py]: Can you please remove this dummy method, use for tests only.

Comment (nadazaric) [reply zanzibar/app.py]: Sure! :)

File: zanzibar/values.py
Comment (tince250) [inline zanzibar/values.py:1]: Maybe rename the file to config.py, to follow the most used convention :)

Comment (nadazaric) [reply zanzibar/values.py:1]: I will change it right away!

**Pull request merged**